microsoft-edge/devtools-guide-chromium/css/grid.md

```diff
@@ -148,9 +144,10 @@ The **Grid overlays** section contains a list of grids that are present on the p
 
 ### Enable overlay views of multiple grids
 
-To display the overlay grid for multiple grids, select the checkbox next to each name of the grid.  In the example, there are 2 grid overlays enabled, that are each represented with different colors:
+To display the overlay grid for multiple grids, select the checkbox next to each name of the grid.  In the example, there are 3 grid overlays enabled, that are each represented with different colors:
```

Comment:
```suggestion
To display the overlay grid for multiple grids, select the checkbox next to each name of the grid.  In the example, there are several grid overlays.  Each grid overlay is represented with a different color, and each grid overlay is enabled:
```
todo: check rendered, to see if there are checkboxes explicitly enabling each grid overlay: could affect pacing/ emphasis in this lead-in
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/css/grid?branch=pr-en-us-3248#enable-overlay-views-of-multiple-grids

Reply:
result:
```
To display the overlay grid for multiple grids, select the checkbox next to each name of the grid:

![Enable overlay views of multiple grids](./grid-images/grid-grid-overlays.png)

In the above example, there are three grid overlays enabled, each grid represented with a different color in the rendered webpage:

*  `body` - the gold grid overlay.
*  `div.fruit-box` - the pink grid overlay.
*  `div.snack-box` - the blue grid overlay.
```